Documentation/network/concepts/masquerading.rst

```diff
@@ -48,6 +48,10 @@ eBPF-based
    file a GitHub issue if you experience any problems. IPv4 BPF masquerading is
    production-ready.
 
+.. note::
+   When BPF masquerading is enabled, Cilium potentially also uses the BPF Host-Routing mode.
+   See :ref:`eBPF_Host_Routing` for benefits and limitations of this mode.
+
 The eBPF-based implementation is the most efficient implementation. It can be enabled with
 the ``bpf.masquerade=true`` helm option.
 
```

Comment:
Can we provide this information without another `.. note::`? There is already a callout note here, and stacking multiple provides poor UX.
```suggestion
The eBPF-based implementation is the most efficient implementation. It can be enabled with
the ``bpf.masquerade=true`` helm option.

By default, BPF masquerading also enables the BPF Host-Routing mode.
See :ref:`eBPF_Host_Routing` for benefits and limitations of this mode.

```